openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -459,10 +458,6 @@ def _delete_cookie(request, response):
     Delete the cookie by setting the expiration to a date in the past,
     while maintaining the domain, secure, and httponly settings.
     """
-    log.warning(
-        u"SafeCookieData is deleting session cookie for user %d",
-        request.user.id
```

Comment:
Of course!  So glad you realized the user wouldn't always be set. I completely missed that in the initial review!

But may there be times when the user is actually available and this could provide useful info?   If so, you can keep it but write it more defensively.

Reply:
I don't think there is, at this point. It was written like this because I wrote it in a different part of the middleware and then moved it here.